app/AppComponents.scala

```diff
@@ -30,12 +30,10 @@ class AppComponents(context: Context, identity: AppIdentity)
 
     override def controllerComponents: ControllerComponents = AppComponents.this.controllerComponents
 
-    override def panDomainSettings: PanDomainAuthSettingsRefresher = new PanDomainAuthSettingsRefresher(
+    override val panDomainSettings: PanDomainAuthSettingsRefresher = PanDomainAuthSettingsRefresher(
       domain = config.pandaDomain,
       system = config.pandaSystem,
-      bucketName = "pan-domain-auth-settings",
-      settingsFileKey = s"${config.pandaDomain}.settings",
-      s3Client = AWS.S3Client,
+      S3BucketLoader.forAwsSdkV1(AWS.S3Client, "pan-domain-auth-settings")
```

Comment:
This PR:

* https://github.com/guardian/pan-domain-authentication/pull/151

...introduced the new `S3BucketLoader` abstraction, which simplifies constructing a `PanDomainAuthSettingsRefresher` and means that Panda is no longer _tied_ to AWS SDK v1 - an alternative AWS SDK v2 implementation of `S3BucketLoader` could be introduced.